snuba/migrations/groups.py

```diff
@@ -17,6 +17,7 @@ class MigrationGroup(Enum):
     SESSIONS = "sessions"
     QUERYLOG = "querylog"
     PROFILES = "profiles"
+    FUNCTIONS = "functions"
```

Comment:
Any reason for this feature to be separate from the profiles group?

Reply:
Migration groups are not contextual to tables, they are logical grouping related to specific product. The reason they exist is to exclude/include a product from migrations entirely.
So it is fundamentally our decision. For your use case I would be ok one way or another.

But when you put two migrations in the same group you are saying that there is a sequential dependency between them. You won't be able to skip one while still running the rest of the group.

Reply:
In this case, I think it makes sense here to separate the migrations into two migration groups so that we can easily make changes to the datasets independently.